app/components/pipeline/teams/Index/chooser.js

```diff
@@ -192,16 +189,6 @@ export default Relay.createContainer(Chooser, {
             }
           }
         }
-        teams(first: 500) @include (if: $isMounted) {
```

Comment:
Awesome :) I think back in the day we got all the teams for the pipeline, and all the organisation pipelines, and just did a diff in JS :)